### PR TITLE
feat(tuto): postman with scaleway API - MTA-1854

### PR DESCRIPTION
**Pull request by ldecarvalho-doc**

### CODE REVIEW

File: tutorials/postman-api/index.mdx [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+4. Click **Save** to do so.
+
+## Setting up authorization 
+
+You can set up authorization at the collection, request category and/or request level. In this tutorial, we will set the authorization at the collection level and configure the requests to inherit the setting from parent. 
+
+1. Go to the **Collections** tab.
+
+2. Click on the Instance API collection. The **Authorization** tab displays by default.
```
Comment (bene2k1) [inline tutorials/postman-api/index.mdx:90]:
```suggestion
2. Click **Instance API collection**. The **Authorization** tab displays by default.
```

**Review by bene2k1 — APPROVED** [review]

File: tutorials/postman-api/index.mdx [hunk 2]
```diff
@@ -0,0 +1,163 @@
+---
+meta:
+  title: How to use Postman to manage Scaleway APIs
```
Comment (scwgoire) [inline tutorials/postman-api/index.mdx:3]:
```suggestion
  title: How to use Postman to use Scaleway APIs
```

File: tutorials/postman-api/index.mdx [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+  description: This page shows how to import APIs into Postman, configure your environment variables and authentication and how to share your workspace with other users. 
+content:
+  h1: Managing Scaleway APIs with Postman
```
Comment (scwgoire) [inline tutorials/postman-api/index.mdx:6]:
```suggestion
  h1: Using Scaleway APIs with Postman
```

File: tutorials/postman-api/index.mdx [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+  paragraph: This page shows how to import Scaleway APIs into Postman, configure your environment variables and authentication and how to share your workspace with other users.  
+tags: postman postman-api scaleway-api postman-download request json
+categories:
+  - storage
```
Comment (scwgoire) [inline tutorials/postman-api/index.mdx:10]: This applies to all Scaleway products, `storage` probably is not the right category here.

Comment (ldecarvalho-doc) [inline tutorials/postman-api/index.mdx:10]:
```suggestion
  - compute
  - network
  - storage
  - iot-hub
```

File: tutorials/postman-api/index.mdx [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+  - compute
+  - network
+  - storage
+  - iot-hub
+dates:
+  validation: 2022-03-24
+  posted: 2022-03-24
+---
+
+Postman is an API client that allows you to build, test and manage APIs through a graphic interface. The platform includes features that make it easy for you to:
```
Comment (RoRoJ) [inline tutorials/postman-api/index.mdx:19]:
```suggestion
Postman is an API client that allows you to build, test and manage APIs through a graphical interface. The platform includes features that make it easy for you to: 
```

File: tutorials/postman-api/index.mdx [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+Postman is an API client that allows you to build, test and manage APIs through a graphic interface. The platform includes features that make it easy for you to: 
+
+  - create, import and test APIs
+  - work collaboratively with other users
+  - create different environments with unique variables
+  - display responses in several programming languages
+  - configure granular request authorization rules, using different authorization types
+  - capture and monitor request data (status, time, size)
+
+In this tutorial you will learn how to import Scaleway APIs into Postman, configure your environment variables and authentication and how to share your workspace with other users.
```
Comment (RoRoJ) [inline tutorials/postman-api/index.mdx:28]:
```suggestion
In this tutorial you will learn how to import Scaleway APIs into Postman, configure your environment variables and authentication and share your workspace with other users. 
```

File: tutorials/postman-api/index.mdx [hunk 7]
```diff
@@ -0,0 +1,166 @@
...
+
+6. Drag and drop the Instances Open API `yml` file into into the indicated space to upload. 
+
+7. Click **Import** to conclude. 
+  
+  The Instances API displays under the **Collections** tab in the side menu. The API is organized into files, following the different request categories (including servers, images, placement groups, IPs).
+
+## Setting up an environment
+
+Postman environments are sets of key/value pairs. They spaces in which you can store your authentication information to use as variables with your collections.
```
Comment (RoRoJ) [inline tutorials/postman-api/index.mdx:66]:
```suggestion
Postman environments are sets of key/value pairs. They are spaces in which you can store your authentication information to use as variables with your collections. 
```

File: tutorials/postman-api/index.mdx [hunk 8]
```diff
@@ -0,0 +1,166 @@
...
+
+7. Click **Import** to conclude. 
+  
+  The Instances API displays under the **Collections** tab in the side menu. The API is organized into files, following the different request categories (including servers, images, placement groups, IPs).
+
+## Setting up an environment
+
+Postman environments are sets of key/value pairs. They spaces in which you can store your authentication information to use as variables with your collections. 
+
+In this section we show you how to create an environment with environment variables for a Scaleway project.
```
Comment (RoRoJ) [inline tutorials/postman-api/index.mdx:68]:
```suggestion
In this section, we show you how to create an environment with environment variables for a Scaleway project.
```

File: tutorials/postman-api/index.mdx [hunk 9]
```diff
@@ -0,0 +1,166 @@
...
+  ```
+6. Click **Send**.
+
+  <Lightbox src="scaleway-postman-response.webp" />
+
+You can **Save** to make sure your configuration is stored for future use.
+
+## Going further
+
+Postman also disposes of a Command Line Interface (CLI) called `newman` that you can use to run your collections. Find out more about newman in the [README](https://github.com/postmanlabs/newman).
```
Comment (RoRoJ) [inline tutorials/postman-api/index.mdx:164]:
```suggestion
Postman also has a Command Line Interface (CLI) called `newman` that you can use to run your collections. Find out more about newman in the [README](https://github.com/postmanlabs/newman).
```

**Review by bene2k1 — APPROVED** [review]

**Review by RoRoJ — APPROVED** [review]

**Pull request merged**